xclim/core/indicator.py

```diff
@@ -1131,15 +1153,18 @@ def _format(
             elif isinstance(v, (int, float)):
                 mba[k] = "{:g}".format(v)
             # TODO: What about InputKind.NUMBER_SEQUENCE
+            elif k == "indexer":
+                if v and v not in [_empty, _empty_default]:
+                    dk, dv = v.copy().popitem()
+                    if dk == "month":
+                        dv = f"m{dv}"
+                    elif dk in ("doy_bounds", "date_bounds"):
+                        dv = f"{dv[0]} to {dv[1]}"
+                    mba["indexer"] = dv
+                else:
+                    mba["indexer"] = args.get("freq") or indices.generic.default_freq()
```

Comment:
Woups, I realize the mistake. `v` is not a correct indexer, that's the point!

But then, I'm not sure of the need of `default_freq`? 
```suggestion
                    mba["indexer"] = args.get("freq") or "YS"
```